.github/workflows/latest_dependency_check.yml

```diff
@@ -8,34 +8,39 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        libraries: ["core", "dask", "koalas"]
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python 3.8
       uses: actions/setup-python@v2
       with:
         python-version: '3.8.x'
-    - name: Install woodwork dependencies and test dependencies
+    - name: Install pip and virtualenv
       run: |
         python -m pip install --upgrade pip
+        python -m pip install virtualenv
+    - name: Update latest core dependencies
+      run: |
+        python -m virtualenv venv_core
+        source venv_core/bin/activate
```

Comment:
I'd add a `pip install --upgrade pip` after activating each virtual env, to ensure the pip inside the virtual env is also up to date

Reply:
Fixed:
https://github.com/alteryx/woodwork/pull/825/commits/2aad95d21414e4413118e7eebc354c8a9e1ad0bf